stable/prometheus-cloudwatch-exporter/templates/deployment.yaml

```diff
@@ -24,6 +24,10 @@ spec:
     spec:
       containers:
         - name: {{ .Chart.Name }}
+          {{- if .Values.logging.enabled }}
+          command: [ "java", "-Djava.util.logging.config.file=/config/logging.properties", "-jar", "/cloudwatch_exporter.jar"]
```

Comment:
I'd rather add an option to customise the command and args than to adding this full line here